### PR TITLE
Standardize terraform resource names

### LINKED ISSUE #105: Standardize terraform resource names
use underscores! (see https://www.terraform-best-practices.com/naming)

...also look for any other opportunities for best practices.

Is there a linter?

### PR DESCRIPTION
**Pull request by davidstanke**
Fixes #105

### CODE REVIEW

**Comment by davidstanke** [comment]
This is a lot of individual changes, but all are simple naming refactors.

File: experimental/terraform/setup.sh [hunk 1]
```diff
@@ -35,7 +35,7 @@ PARENT_PROJECT=$(gcloud config get-value project 2>/dev/null)
 if [[ ${AUTO} == 'true' ]]
 then
     # populate setup variables (for use in testing/dev)
-    make_new_project="y"
+    make_new_project='y'
```
Comment (dinagraves) [inline experimental/terraform/setup.sh:38]: Why this change?  File seems to have a mix of double and single quotes.

**Review by dinagraves — APPROVED** [review]

File: experimental/terraform/setup.sh (hunk 1 above)
Comment (davidstanke) [reply experimental/terraform/setup.sh:38]: ...and, um... it still has a mix! I was trying to find an error and noticed that this variable was set with double quotes, but then compared later to a literal with single quotes. I didn't think that would make a difference, but you know. So I made it consistent. Well, this one's consistent. The overall file isn't! 

It may be useful to apply a bash style guide... do you recommend one?
